### PR TITLE
docs: add multisig documentation

### PR DESCRIPTION
**Pull request by Josumner**

### CODE REVIEW

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ theme:
   feature:
     tabs: true
 
-strict: true
+strict: false
```
Comment (ejfitzgerald) [inline docs/mkdocs.yml:12]:
```suggestion
strict: true
```

Lets restore this back to the original

File: docs/docs/cli-multisig.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Multisig keys
+Documentation describing the creation of multisig keys and signing transactions.
```
Comment (ejfitzgerald) [inline docs/docs/cli-multisig.md:2]:
```suggestion
```

I think that we simply remove this line

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]
Really nice one @Josumner , just a couple of minor updates and we should get this merged

File: docs/docs/cli-multisig.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+This feature of `fetchd` allows users to securely control keys in a number of configurations. Using a threshold number K of maximum N keys, a user or group of users can set the minimum number of keys required to sign a transaction. Some examples of these configurations allow some useful features such as the choice of a spare key, where only one key is required to sign (K=1) but there are two keys available to do so. Another more complex example configuration is set out below.
+
+## Creating a multisig key
+The following represents the syntax and argument layout of the `fetchd` command to create a multisig key.
+
+```
+// Create a simple multisig key with a threshold of 1 as default
```
Comment (daeMOn63) [inline docs/docs/cli-multisig.md:10]:
```suggestion
# Create a simple multisig key with a threshold of 1 as default
```
an extra nit: comments in bash are with a `#`, not `//`

**Review by ejfitzgerald — APPROVED** [review]
Nice one!

**Pull request merged**